### PR TITLE
Docker build

### PR DESCRIPTION
**Pull request by LennardBoediger**
all test passed 
https://git.dotplex.com/nitrokey/nitrokey-storage-firmware/-/pipelines

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:latest
+RUN apt-get update && apt-get install unzip make wget binutils-avr git -qy
+# COPY avr32studio-ide-2.6.0
+ARG TOOLCHAIN_LINK=https://cloud.nitrokey.com/s/a4Z2abXRjjYEzDJ/download
+RUN cd / && wget ${TOOLCHAIN_LINK} -O toolchain.zip -q && \
+    unzip toolchain.zip && \
```
Comment (szszszsz) [inline Dockerfile:6]: Can we make unzip silent? It clutters the view.

**Review by szszszsz — COMMENTED** [review]
Looks good! Let's discuss this one thing.

File: Dockerfile (hunk 1 above)
Comment (LennardBoediger) [reply Dockerfile]: done

**Comment by szszszsz** [comment]
Hi! Let me know once you are ready. Can you switch to draft in the meantime?
Please check the typo in the commit message.

**Comment by LennardBoediger** [comment]
it should be ready to be merged, but there are some server issues that prevent me from doing the final test, so still on hold

**Comment by szszszsz** [comment]
Hi! I see you are still changing the PR. Let me know once done!

**Comment by LennardBoediger** [comment]
the artifacts path was wrong - now it is really ready to merge

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - if: '$CI_PIPELINE_SOURCE == "web"'
+  tags:
+    - lxc
+  stage: build
+  script:
+    - cd src && make build-docker-toolchain
+    - make docker-make
+  after_script:
+    - cp ./src/*.hex artifacts
+    - wget 188.40.174.112:8089/checkmark/$CI_COMMIT_REF_NAME/$CI_COMMIT_SHA/$CI_JOB_NAME/$CI_JOB_STATUS/${CI_JOB_URL#*/*/*/}
```
Comment (szszszsz) [inline .gitlab-ci.yml:31]: Is this IP documented somewhere? Why IP instead of a domain (not futureproof, as IPs are changing)?

**Review by szszszsz — APPROVED** [review]
Please add the documentation for the mentioned IP asap.
Merging.

**Pull request closed without merging**